test/unit/org/apache/cassandra/dht/SplitterTest.java

```diff
@@ -490,24 +490,6 @@ public void testPositionInRangeMurmur3Partitioner()
         testPositionInRangeMultiRange(new Murmur3Partitioner());
     }
 
-    // STAR-1590
```

Comment:
Why is this being removed?

Reply:
Oh, now I remember - because CASSANDRA-18013 added `testSplitOwnedRanges` that is the identical test as `testSplitOwnedRangesForSmallRange` in STAR-1590.

I think that CASSANDRA-18013 is the same change as STAR-1590.  I decided to prefer the C* code over the STAR code change.